src/gemdat/plots/matplotlib/_paths.py

```diff
@@ -7,28 +7,42 @@
 from gemdat.volume import Volume
 
 
-def energy_along_path(*, path: Pathway, structure: Structure,
+def energy_along_path(*, paths: Pathway | list[Pathway], structure: Structure,
```

Comment:
Could be a clever use for *paths here. Not sure, just a thought.

```suggestion
def energy_along_path(*paths: Pathway | list[Pathway], *, structure: Structure,
```

Reply:
Not sure about this, I think if I prefer *, paths for clarity. Is there a technical advantage that you see?

Reply:
The technical advantage is that with `*paths`, the paths variable is always a list, so you don't need to do any instance checking. It's not super common though so it's a bit awkward to use sometimes. I also think `*, paths` is clearer if documented well.